src/components/addmyriadataset/selectize.default.scss

```diff
@@ -0,0 +1,442 @@
+/**
```

Comment:
I don’t think this file should be here.  

Other directives can use selectize too.  So we better put this in asset?